### DISCUSSION TITLE
Can't add extensibility libraries to load

### DISCUSSION THREAD

**Discussion by Luca3082** [microsoft-search/pnp-modern-search#2935] [Q&A]
**Version used**
Ex: 4.7

**Describe the bug**
'm using PnP Search Results and I create a library to extends the search results. When I try to add Extensibility libraries to load, the webpart doesn't add nothing at all. This is strange because I'm developing and testing my library for almost 2 weeks without any problems.
I noticed that when i click to Configure button and the panel appears, when i click to the Manifest GUID textbox to add a value, in Chrome Console a series of errors like this:

POST https://dc.services/visualtudio.com/v2/track 439

And when i click to Save button the panel closed without saving...

**To Reproduce**
Detailed steps to reproduce the behavior:
1. Go to PnP Search Results, fourth step of configuration
2. Click on Configure under Extensibility libraries to load
3. Add a new Manifest GUID
4. Click on Save
5. Go back to the panel clicking Configure again and the value is not saved

**Screenshots**

![image](https://user-images.githubusercontent.com/15248198/234911894-f450e8fb-1d0e-425f-b7af-6854708de4c1.png)

![image](https://user-images.githubusercontent.com/15248198/234912052-e084a2fe-614d-4185-9c1b-2a4fcb477dbe.png)

![image](https://user-images.githubusercontent.com/15248198/234912225-7cf9f232-182c-4a7b-b967-eba055e778bc.png)

**Desktop (please complete the following information):**
 - Chrome

Thanks in advance

**Comment by wobba** [comment]
Have you made sure the ext lib is built with the same SPFx version as the web part version you use?

**Reply by Luca3082** [reply]
yes, the thing is that i already use this ext lib for almost 2 weeks without problems, adding new components correctly. suddenly that happens...
telemetry problem obviously is not the point. I disable telemetry so the 439 error doesn't appear anymore, but still, I can't add correctly my guid to the webpart...

**Reply by wobba** [reply]
If you didn't change anything and the issue suddenly appears on a page, that would be a SPFx issue then and not part releated(?)
